docs/source/tutorials/mint-your-first-asset.md

```diff
@@ -0,0 +1,90 @@
+# Minting your first Asset
+
+When you use Iroha in your application you will definitely face the need to mint an asset.
+Use this tutorial to master Iroha Special Instructions in general and `MintAsset` in particular.
+
+## Prerequisites
+
+* [Rust](https://www.rust-lang.org/tools/install)
+* Text Editor or IDE
+* Existing Rust project //TODO: create some sandbox project
+
+## Steps
+
+### 1. Add dependency on Iroha crate
+
+Inside `path_to_your_project/Cargo.toml` add a new dependency:
+
+```toml
+iroha = "2.0.0"
+```
+
+### 2. Find a place where you need to mint an asset
+
+Usually any business operation will have a need to mint an asset:
+
+* Uploading of digital documents
+* Withdraw of crypto currencies
```

Comment:
`Withdrawal`

Reply:
And also why withdrawal? I would add generating an initial supply of cryptocurrency as example.